pelicun/file_io.py

```diff
@@ -282,26 +286,53 @@ def substitute_default_path(
       are located.
     - If a path in the input list does not contain 'PelicunDefault/',
       it is added to the output list unchanged.
+    - If the file after 'PelicunDefault/' does not exist in the
+      `resource_paths` keys, a `KeyError` is raised.
 
     Examples
     --------
-    >>> data_paths = ['PelicunDefault/data/file1.txt',
-        'data/file2.txt']
+    >>> data_paths = ['PelicunDefault/data/file1.txt', 'data/file2.txt']
     >>> substitute_default_path(data_paths)
-    ['{base.pelicun_path}/resources/SimCenterDBDL/data/file1.txt',
+    ['{base.pelicun_path}/resources/SimCenterDBDL/updated_path/file1_v2.txt',
```

Comment:
Please update this example to use the new path name (i.e., with DamageAndLossModelLibrary)

Reply:
Good catch, thanks.